--- gcc-arm-none-eabi-4_7-2013q3-20130916/build-common.sh	2013-09-15 12:19:48.000000000 -0700
+++ gcc-arm-none-eabi-4_7-2013q3-20130916.new/build-common.sh	2013-11-05 14:26:46.436225855 -0800
@@ -329,8 +329,8 @@
 # on Ubuntu and Mac OS X.
 uname_string=`uname | sed 'y/LINUXDARWIN/linuxdarwin/'`
 if [ "x$uname_string" == "xlinux" ] ; then
-    BUILD=i686-linux-gnu
-    HOST_NATIVE=i686-linux-gnu
+    BUILD=x86_64-linux-gnu
+    HOST_NATIVE=x86_64-linux-gnu
     READLINK=readlink
     JOBS=`grep ^processor /proc/cpuinfo|wc -l`
     GCC_CONFIG_OPTS_LCPP="--with-host-libstdcxx=-static-libgcc -Wl,-Bstatic,-lstdc++,-Bdynamic -lm"
