--- gcc-arm-none-eabi-4_7-2013q3-20130916/build-prerequisites.sh	2013-09-15 12:19:48.000000000 -0700
+++ gcc-arm-none-eabi-4_7-2013q3-20130916.new/build-prerequisites.sh	2013-11-05 15:03:54.032082305 -0800
@@ -162,7 +162,6 @@
     --target=$TARGET \
     --prefix=$BUILDDIR_NATIVE/host-libs/usr \
     --disable-shared \
-    --disable-nls \
     --with-host-libstdcxx='-lstdc++'    \
     --with-gmp=$BUILDDIR_NATIVE/host-libs/usr \
     --with-ppl=$BUILDDIR_NATIVE/host-libs/usr
