diff -ruN a/build-toolchain.sh b/build-toolchain.sh
--- a/build-toolchain.sh	2014-02-16 09:25:13.033064431 -0800
+++ b/build-toolchain.sh	2014-02-16 09:27:39.829067187 -0800
@@ -326,7 +326,7 @@
 for d in $plugin_src_dirs; do
     plugin_name=$(basename $d)
     src_files=$(find $d -name \*.c -or -name \*.cc)
-    g++ -fPIC -fno-rtti -O2 -shared -I $plugin_dir/include $src_files \
+    g++ -fPIC -fno-rtti -O2 -shared -I $plugin_dir/include -I $BUILDDIR_NATIVE/host-libs/usr/include $src_files \
       -o $plugin_dir/$plugin_name.so
 done
 fi
